README.md

```diff
@@ -118,7 +132,7 @@ Then it can be used as follows:
 ./lib/linux/x86_64/libpv_cheetah.so \
 ./lib/common/acoustic_model.pv \
 ./lib/common/language_model.pv \
-./resources/license/cheetah_eval_linux_public.lic \
+./PATH_TO_YOUR_CHEETAH_LICENSE_FILE \
```

Comment:
remove "./" please. Also we used ${PATH_TO_YOUR_CHEETAH_LICENSE_FILE} notation for variables in shell across documentation.